src/libdredd/CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ add_library(
   include/libdredd/mutation_remove_statement.h
   include/libdredd/mutation_replace_binary_operator.h
   include/libdredd/new_mutate_frontend_action_factory.h
+  include/libdredd/mutation_replace_unary_operator.h
```

Comment:
Please keep files in alphabetical order (so move this up).

Reply:
Done.